config.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "cert": "/Users/x/usercert.pem",
+        "ckey": "/Users/y/userkey.pem",
+        "keytab": ""
```

Comment:
you do not need to list keytab if it is empty, and vice versa if keytab is provided you do not need cert/ckey